### PR TITLE
Designing the about us section and styling add imgs

### LINKED ISSUE #4: About us section
Creating about us section and styling it

### PR DESCRIPTION
**Pull request by RandSohail**
Relates #4
design the html structure for about us section 
make the css styling 
add 2 imgs

### CODE REVIEW

**Review by abdulrahman-2020 — APPROVED** [review]

**Review by ahmadfarid-059 — APPROVED** [review]

**Review by abdulrahman-2020 — APPROVED** [review]

File: style.css [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+}
+/* ---------------- ABOUT US CONTAINER -------------------*/
+#about-us-container {
+  background-color: #f9f9f9;
+  width: 80%;
+  height: 400px;
+  border-radius: 15px;
+  border: #c9c9c9 solid 0.2px;
+  display: grid;
+  grid-template-columns: 50% 50%;
```
Comment (Ibrahim-Jarada) [inline style.css:37]: I prefer if you can try to use flex instead grid
~~display: grid;
  grid-template-columns: 50% 50%;~~
```
display: flex;

  justify-content: space-evenly;

  align-items: center;

  align-content: center;
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  margin: 0 auto;
+}
+
+/* ------------------------- ABOUT US LEFT SECTION ------------------*/
+
+.about-us-left-section {
+  position: relative;
+  min-height: 1px;
+  padding-left: 15px;
```
Comment (Ibrahim-Jarada) [inline style.css:46]: Try add max width to your section to enhance view and styling
`    max-height: 100%;
`

File: style.css [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+}
+
+.about-us-computer-img {
+  max-width: 100%;
+  transform: translateY(-50%);
```
Comment (Ibrahim-Jarada) [inline style.css:51]: There the is no need for this line you can style your element without it
~~transform: translateY(-50%);~~

File: style.css [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+  z-index: 1;
+}
+.about-us-ipad-img {
+  max-width: 50%;
```
Comment (Ibrahim-Jarada) [inline style.css:55]: After the suggested modification you'll need to change this value
~~max-width: 50%;~~
`  max-width: 25%;
`

File: style.css [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+  max-height: 50%;
```
Comment (Ibrahim-Jarada) [inline style.css:56]: You can remove this value
~~max-height: 50%;~~

File: style.css [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+  z-index: 2;
+  position: relative;
+  transform: translateY(80%);
```
Comment (Ibrahim-Jarada) [inline style.css:59]: You can replace these values
~~position: relative;
transform: translateY(80%);~~
By
```
position: absolute;
  left: 20%;
  bottom: 0;
```

File: style.css [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+}
+
+/* ------------------------- ABOUT US LEFT SECTION ------------------*/
```
Comment (Ibrahim-Jarada) [inline style.css:62]: Typo error
ABOUT US ~~LEFT~~ RIGHT SECTION

**Review by Ibrahim-Jarada — COMMENTED** [review]
You have overflow problem
Your section interferes with footer.
So Facebook icon and LinkedIn not clickable anymore.
Try to enhance your style

I'll share with you the way I style, you may do even better

**Pull request merged**